### PR TITLE
WOR-1156 fix incorrect api docs

### PR DESCRIPTION
**Pull request by blakery**
Ticket: [WOR-1156](https://broadworkbench.atlassian.net/browse/WOR-1156)

The swagger doc was incorrect. This brings it in line with what the service is actually expecting.
* add protectedData property to correct component definition 
* remove unused component
---

**PR checklist**

- [ ] Include the JIRA issue number in the PR description and title
- [ ] Make sure Swagger is updated if API changes
  - [ ] **...and Orchestration's Swagger too!**
- [ ] If you changed anything in `model/`, then you should [publish a new official `rawls-model`](https://github.com/broadinstitute/rawls/blob/develop/README.md#publish-rawls-model) and update `rawls-model` in [Orchestration's dependencies](https://github.com/broadinstitute/firecloud-orchestration/blob/develop/project/Dependencies.scala).
- [ ] Get two thumbsworth of PR review
- [ ] Verify all tests go green, including CI tests
- [ ] **Squash commits and merge** to develop (branches are automatically deleted after merging)
- [ ] Inform other teams of any substantial changes via Slack and/or email

[WOR-1156]: https://broadworkbench.atlassian.net/browse/WOR-1156?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: core/src/main/resources/swagger/api-docs.yaml [hunk 1]
```diff
@@ -5608,44 +5608,6 @@ components:
           items:
             $ref: '#/components/schemas/Entity'
       description: ""
-    CreateRawlsBillingProjectFullRequest:
```
Comment (blakery) [inline core/src/main/resources/swagger/api-docs.yaml:5611]: This no longer exists anywhere in the project, or elsewhere in this file

File: core/src/main/resources/swagger/api-docs.yaml [hunk 2]
```diff
@@ -5608,44 +5608,6 @@ components:
...
-            VPC network to only allow access to GCP APIs that are protected by the
-            project's service perimeter and routes all allowed API traffic through
-            a narrow IP range. Requires highSecurityNetwork = true.
-        servicePerimeter:
-          type: string
-          description: The fully qualified name of the GCP service perimeter to put
-            this project into in the form accessPolicies/[POLICY NUMBER]/servicePerimeters/[NAME].
-            Caller must have the add_project action for this service perimeter in
-            Sam.
-        protectedData:
```
Comment (blakery) [inline core/src/main/resources/swagger/api-docs.yaml:5643]: This was added to the wrong component

**Review by aherbst-broad — APPROVED** [review]
Thanks for catching this.

**Review by cahrens — APPROVED** [review]
Does orchestration Swagger also need to be updated?

**Pull request merged**